spring-data/function-invocation/start.gfsh

```diff
@@ -0,0 +1,4 @@
+start locator
+configure pdx --auto-serializable-classes=dev\.gemfire\.function\.domain\..*
+start server
+deploy --jar=./build/libs/function-invocation-plain.jar
```

Comment:
I thought we had spoken about splitting this out as a separate step? Also there is no verification step in the README that aids the user in confirming that the functions have successfully been deployed

Reply:
We'd discussed that, and we'd also discussed having the user cd into the example directory, in which case this should work–I thought we were doing the second option. I can break it out though regardless.